Makefile.am

```diff
@@ -2,6 +2,10 @@ configdir = $(datarootdir)/${PACKAGE_NAME}
 AM_CFLAGS = -g -Wall -D_GNU_SOURCE -D 'CONFIG_PATH="${configdir}"' -I$(srcdir)/include
 ACLOCAL_AMFLAGS = -I config
 
+if MACOS
+os_excludes = -e msg_epoll
+endif
```

Comment:
Having an exclusion file for tests, then still relying on makefile for additional exclusions is odd.  Exclusions should be in one place, not spread in multiple places.

Reply:
Agreed but this test wouldn't run on OSX at all. If we're to have it in the test exclusion file, we would have to add it in every provider's test exclusion file and also maintain different versions of the file for Linux and OSX. Specifying it here in one place excludes this test on OSX for any provider.

Caveat: if an user is running runfabtests.sh manually instead of `make test` on OSX they still have to specify `-e msg_epoll` on the command line. I don't like this but the alternative is to maintain multiple test exclusion files for different OS or supporter richer format for the exclusion file which would increase parsing complexity.

Reply:
Can we allow specifying multiple exclusion files, so we can pass in a provider file plus an OS file?

Reply:
> Can we allow specifying multiple exclusion files, so we can pass in a provider file plus an OS file?

I think this should be possible.